dev-util/cutter/cutter-2.0.1.ebuild

```diff
@@ -42,7 +44,8 @@ src_configure() {
 		-DCMAKE_CXX_COMPILER="$(tc-getCXX)"
 		-DCMAKE_C_COMPILER="$(tc-getCC)"
 		-DCMAKE_INSTALL_PREFIX="${EPREFIX}/usr"
-		-DCUTTER_ENABLE_GRAPHVIZ=OFF
+		-DCUTTER_USE_ADDITIONAL_RIZIN_PATHS=OFF
```

Comment:
Keep the options sorted alphabetically, please

Reply:
ok